assets/config.yml

```diff
@@ -2426,6 +2426,12 @@ services:
   - name: "Security & Protection"
     icon: "fa-solid fa-shield"
     items:
+
+        - name: "SpyShelter Antispyware"
```

Comment:
This tabulation causes the page not to load correctly. 
![image](https://github.com/user-attachments/assets/859cdede-ef58-4446-9a18-aedcf994ac83)

This should be 
```
  - name: "Security & Protection"
    icon: "fa-solid fa-shield"
    items:
      - name: "SpyShelter Antispyware"
        logo: "assets/tools/spyshelter.png"
        url: "https://www.spyshelter.com"
        target: "_blank" # optional html a tag target attribute
```

Reply:
I apologize.  Thank you for fixing the issue and it looks like it is resolved on the website.